src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -180,6 +180,7 @@ function UserPermissionsPopUp({ allUserProfiles, getAllUsers, roles, authUser, d
             rolePermissions={userPermissions}
             immutablePermissions={actualUserRolePermission}
             editable={!!actualUserProfile}
+            useCase="editRole"
```

Comment:
This popup is not for editing roles. This is for editing individual user permissions. Need to swap this to the correct use case/permission and find the other file for editing roles to add this there.